ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -3,9 +3,13 @@ let prototyped_of_class = function
       None
 
 let prototyped_of_field = function
+  | "Repository", "gpgkey_path" ->
+      Some "22.10.0.4.gbf8e4e52c-next"
```

Comment:
Hmm... this should've been `22.10.0-next`. So I made a mistake in my PR about versions.